custom/export_xml.php

```diff
@@ -252,25 +255,36 @@ function addInsurance($row, $seq) {
 ?>
 <html>
 <head>
-<?php html_header_show();?>
-<link rel=stylesheet href="<?php echo $css_header;?>" type="text/css">
-<title><?php xl('Export Patient Demographics','e'); ?></title>
+<?php Header::setupHeader(); ?>
+<title><?php echo xlt('Export Patient Demographics'); ?></title>
 </head>
 <body>
-
-<p><?php xl('The exported data appears in the text area below.  You can copy and
-paste this into an email or to any other desired destination.','e'); ?></p>
-
-<center>
 <form>
+  <div class="container">
+     <div class="row">
+        <div class="col-xs-12">
+           <div class="page-header">
+              <h4><?php echo xlt('The exported data appears in the text area below. You can copy and paste this into an email or to any other desired destination.'); ?></h4>
+           </div>
+        </div> 
+     </div>
+     <div class="row">
+        <div class="col-xs-12">
+           <div class="form-group">
+              <textarea name="export_data" class=form-control cols="50" rows="18"><?php echo text($out) ?></textarea>
+           </div>
+           <div class="form-group">
+              <div class="col-sm-offset-2 col-sm-10">
+                 <div class="btn-group" role="group">
+                    <button type="button" class="btn btn-default btn-save" onclick="window.close()"><?php echo xlt("OK"); ?></button>
```

Comment:
I think better text would be "Close this tab" along with a negative-action icon (`btn-cancel` instead of `btn-save`)